services/ui_backend_service/api/utils.py

```diff
@@ -11,12 +11,17 @@
 from services.data.db_utils import DBPagination, DBResponse
 from services.utils import format_baseurl, format_qs, web_response
 from functools import reduce
+from services.utils import logging
+
+
+logger = logging.getLogger("Utils")
 
 
 def get_json_from_env(variable_name: str):
     try:
         return json.loads(os.environ.get(variable_name))
-    except Exception:
+    except Exception as e:
+        logger.warning("error parsing JSON: %s, from %s: %s", e, variable_name, os.environ.get(variable_name))
```

Comment:
Excellent info with the added logging. The old style string interpolation is mainly used for passing string templates around on the service side, so for consistency the warning formatting could instead use .format() or an f-string, whichever is more readable? Up to you if the change makes sense to implement :)

Reply:
consider specifically catching json parsing errors instead though, as we usually _try_ to load configs from env, but most of the time it is not an exception if an env_var is not set, so we probably do not want a warning in these cases.

Reply:
consider specifically catching json parsing errors instead though, as we usually try to load configs from env, but most of the time it is not an exception if an env_var is not set, so we probably do not want a warning in these cases.